free-1.md

```diff
@@ -10,41 +10,79 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Since Twitch Plays Pokemon began in 2014, there has been lots of interest
+in creating similar "crowdplay" games. However, it is an area primarily
+done by people willing to overcome the steep curve of learning how to interface
+with emulators and Twitch's API. A DSL could help people who would have previously
+been excluded from making a "Twitch Plays"
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL is appropriate for multiple reasons. We are focusing on a specific domain,
+listening and executing Twitch chat commands. In addition, we are attempting to
+make that one domain extremely easy while allowing for everything else to be
+hard or impossible.
+
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I was a big fan of the original Twitch Plays Pokemon and I was excited to see
+other people continue the tradition, but no one did. I attribute this
+significantly to the difficulty in creating such a project.
+
 
 ### Domain
 _Describe the project's domain in five words._
 
+Crowdsourcing video gameplay via Twitch.
+
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+I was imagining the language as a declarative one. There is a lot of repetitive
```

Comment:
Declarative language seems like the right way to go, but I wonder how you will be able to create a degree of fluency and differentiate it from acting just as an API.